graphql/internal/resolver/resolver.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/abbeyhrt/keep-up/graphql/internal/models"
 	"github.com/abbeyhrt/keep-up/graphql/internal/session"
 	graphql "github.com/graph-gophers/graphql-go"
+	"github.com/opentracing/opentracing-go/log"
```

Comment:
I think this might be the wrong `log` package. Might have been imported accidentally 🤔